src/lib/containers/ProgrammaticInstructionsModal.tsx

```diff
@@ -73,10 +73,8 @@ export const ProgrammaticInstructionsModal = ({
       size="lg"
     >
       <Modal.Header closeButton>
-        <Modal.Title>
-          <Typography variant={'headline1'}>{title}</Typography>
-          {helpMarkdown && <HelpPopover markdownText={helpMarkdown} helpUrl={helpUrl} />}
-        </Modal.Title>
```

Comment:
I think `Modal.Title` is important for getting the correct sizing and aligning the close button. Any particular reason you removed it?

Reply:
Good point.  The Modal Title style was being applied to the HelpPopover (and child contents), meaning the popover help text was huge.  I'll try wrapping the HelpPopover text in body1 Typography instead